.github/actions/checkout-adobe/action.yml

```diff
@@ -0,0 +1,55 @@
+name: 'Checkout Adobe RMSDK'
```

Comment:
A general action we can reference in our other builds, to checkout the new RMSDK repo, with some caching, so we don't have to download the whole thing every time.